src/middlewares/validations.js

```diff
@@ -4,10 +4,13 @@ import customMessages from '../utils/customMessages';
 /**
 * @param {string} pattern
 * @param {string} messages
+* @param {boolean} isProfileUpdate
 * @returns {object} a chained validation methods
 */
-const validationMethods = (pattern, messages) => {
```

Comment:
can you move this function into another file `/src/middlewares/validationMethods.middleware.js`, so that if anyone else needs to use this validations but with different pattern and messages, can just call your code and pass in their pattern and messages.

Reply:
yes sure, I move that file to utils